ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml

```diff
@@ -768,6 +768,7 @@ tests:
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
       COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      E2E_RUN_TAGS: not @workloads
```

Comment:
Do we need this for 4.16, too ?

BTW, could you explain more details ?  I'd like to know which capability cause this test filter requirement, so that I can check if it can be auto detected.

Reply:
@liangxia i do not see build and dc capability enabled due to which tests dealing with build, buildconfig & deploymentconfig are failing, so added it.

Reply:
For this profile, the build and dc capability are not always disabled, as the capability set to disabled are choose randomly. Always skip workloads tests are not expected as we have this capability on this profile sometimes.
/cc @jinyunma

Reply:
BTW, I've implemented a dynamic way to include/exclude workloads tests based on the capability on the cluster in https://github.com/openshift/release/pull/49946

Reply:
yes, this profile is not always disable one specific capability, but choose one randomly to disable. 
Thanks @liangxia, dynamic way should fix the issue.

Reply:
@liangxia @jinyunma thank you !!